operator/pkg/secrets/secrets_test.go

```diff
@@ -51,6 +51,36 @@ func TestMaybeExpand(t *testing.T) {
 	}
 }
 
+func TestExpand(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name           string
+		value          string
+		ignoreNotFound bool
+		expectedError  bool
+		expectedString string
+	}{
+		{
```

Comment:
Mind adding a case where ignoreNotFound is true to so both sides of this logic get exercised?